<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>handleSuggestionDialog</class>
 <widget class="QDialog" name="handleSuggestionDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="cardSelectionBox">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>190</y>
     <width>156</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="suggestionInfo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>281</width>
     <height>106</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="okayButton">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>190</y>
     <width>97</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Okay</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>145</y>
     <width>286</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Please select one of your cards to reveal to disprove the suggestion.</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
